lib/util/exec/docker/index.ts

```diff
@@ -258,3 +259,20 @@ export async function generateDockerCommand(
 
   return result.join(' ');
 }
+
+export async function deleteCacheVolume(): Promise<void> {
+  const { dockerChildPrefix } = getAdminConfig();
+  const volumeNamespace = getContainerName('manager_cache', dockerChildPrefix);
```

Comment:
Only if docker mode, otherwise it fails on global mode, when no docker is available.

We should also add a new config option, as maybe not all users want those docker volumes, or override by some more persistent.

Reply:
Sure, it's just a blueprint